render-test/runner.cpp

```diff
@@ -46,7 +47,236 @@ const std::vector<std::string>& TestRunner::getPlatformExpectationsPaths() {
     return result;
 }
 
-bool TestRunner::checkResults(mbgl::PremultipliedImage&& actualImage, TestMetadata& metadata) {
+// Strip precision for numbers, so that we can compare evaluated results with fixtures.
+// Copied from JS expression harness.
+Value stripPrecision(const Value& value) {
```

Comment:
could we share common code between expression and render runners?

Reply:
Done: https://github.com/mapbox/mapbox-gl-native/pull/15770